multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/PollExecuteTaskStatusStepTest.java

```diff
@@ -85,6 +90,61 @@ void testPollStateExecution(CloudTask.State currentTaskState, long currentTime,
         testExecuteOperations();
     }
 
+    @Test
+    void testFirstTimeOnlyInitializesTimestamp() {
+        initializeParameters(CloudTask.State.PENDING, System.currentTimeMillis());
+
+        context.setVariable(Variables.LAST_TASK_POLL_LOG_TIMESTAMP, null);
+        this.expectedExecutionStatus = AsyncExecutionState.RUNNING;
+
+        testExecuteOperations();
+
+        Long timeStamp = context.getVariable(Variables.LAST_TASK_POLL_LOG_TIMESTAMP);
+        assertNotNull(timeStamp);
```

Comment:
assertions for not null are not necessary when you have another assert which asserts for different thing.